### PR TITLE
comments

### PR DESCRIPTION
**Pull request by ineiti**
Comments...

### CODE REVIEW

File: protocols/cosi/packets.go [hunk 1]
```diff
@@ -44,8 +44,7 @@ type ProtocolPacket struct {
 // ProtocolIO implements the sda.ProtocolIO interface for the CoSi protocol.
 type ProtocolIO struct{}
 
-// Wrap implements the sda.ProtocolIO interface by wrapping up any of the
-// four-step messages into a ProtooclPacket.
+// Wrap takes a dynamic OverlayMessage and returns a ProtocolPacket.
```
Comment (nikkolasg) [inline protocols/cosi/packets.go]: what is a "dynamic" message ? I would not put dynamic unless we have a clear definition of what it is exactly. Same for the other occurences.

File: protocols/cosi/packets.go [hunk 2]
```diff
@@ -94,33 +92,31 @@ func (p *ProtocolIO) Unwrap(msg interface{}) (interface{}, *sda.OverlayMessage,
...
 }
 
-// Response with which every node replies with.
+// Response of all nodes, aggregated over all children.
 type Response struct {
 	Resp abstract.Scalar
 }
 
-//Theses are pairs of TreeNode + the actual message we want to listen on.
+// Overlay-structures to retrieve the sending TreeNode.
```
Comment (nikkolasg) [inline protocols/cosi/packets.go]: Singular

File: protocols/cosi/packets.go (hunk 1 above)
Comment (ineiti) [inline protocols/cosi/packets.go]: It's in the ProtocolIO description.

File: protocols/cosi/packets.go (hunk 2 above)
Comment (ineiti) [inline protocols/cosi/packets.go]: there are four structures that follow.

**Pull request merged**